src/libstore/derivations.cc

```diff
@@ -15,7 +15,8 @@ std::optional<StorePath> DerivationOutput::pathOpt(const Store & store, std::str
         },
         [&](DerivationOutputCAFixed dof) -> std::optional<StorePath> {
             return {
-                store.makeFixedOutputPath(dof.hash.method, dof.hash.hash, drvName)
+                // FIXME if we intend to support multiple CA outputs.
+                dof.path(store, drvName, "out")
```

Comment:
Doesn't this invalidate the hash of existing CA derivations?

Reply:
I think this does exactly what the old code did, just factored out?

Reply:
Ah yes, didn't notice that "out" was special-cased so that `outputPathName(foo, "out") == foo`

Reply:
We since landed https://github.com/NixOS/nix/pull/3875 which should allow doing this better, and including working hypothetical multi-output fixed-output CA derivations by not assuming `"out"`.

Reply:
FIXME is now gone.